cmd/utask/root.go

```diff
@@ -31,12 +31,14 @@ const (
 	defaultInitializersFolder = "./init"
 	defaultPluginFolder       = "./plugins"
 	defaultTemplatesFolder    = "./templates"
+	defaultScriptsFolder      = "./scripts/"
```

Comment:
uneasy about this trailing slash, IMO should be handled in code

Reply:
agree